chicken-sloths/bangazon-api-sprint1
# Deleted extra join table files

# Description
This deletes old files we weren't using. (We created a bunch of files for join table routes before we realized that we'd never be querying join tables directly.) 

## Number of Fixes
Fixes Issue #113 

## Expected Behavior
Files that should be gone:
Controllers
1. ``product_orderCtrl.js``
1. ``employeeComputerCtrl.js``
1. ``employeeTrianingProgramCtrl.js``

Models
1. ``EmployeeComputersModel.js``
1. ``EmployeeTrainingProgramsModel.js``
1. ``ProductOrdersModel.js``

Routes
1. ``routes/employeeComputers.js``
1. ``routes/employeeTrainingPrograms.js``
1. ``routes/productOrders.js``


## Steps to Test Solution

Run a `GET` for each route and make sure they all work!
1. `/employees`
1. `/departments`
1. `/training-programs`
1. `/computers`
1. `/customers`
1. `/products`
1. `/orders`
1. `/payment-options`
1. `/product-types`
